app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt

```diff
@@ -231,6 +229,11 @@ class PatientSummaryEffectHandler @AssistedInject constructor(
     return countOfRecordedBloodPressures + countOfRecordedBloodSugars
   }
 
+  private fun diagnosisRecorded(patientUuid: UUID): Boolean {
+    val medicalHistory = medicalHistoryRepository.historyForPatient(patientUuid)
+    return (medicalHistory.diagnosedWithDiabetes != Unanswered && medicalHistory.diagnosedWithHypertension != Unanswered)
```

Comment:
Can we move this boolean logic into the `MedicalHistory` model as a computed property? That way, this gets defined in one place, and is a property of the domain instead of being in the effect handler.

Reply:
Once we do this, we can also add a few simple unit tests that `diagnosisRecorded` is computed correctly.